src/reducers/MapReducer.js

```diff
@@ -16,9 +46,12 @@ const INITIAL_STATE = {
         latitudeDelta: 1
     },
     mapLoaded: false,
-    viewStyle: map.SearchOptions.MAP
+    viewStyle: map.SearchOptions.MAP,
+    sites: sampleSiteMarkers
```

Comment:
`sampleSiteMarkers` are now part of the initial state. This will go to a blank array when we have a real endpoint to hit.